__init__.py

```diff
@@ -8,6 +8,7 @@
 import boto3
 import shutil
 import subprocess
+import rembg
```

Comment:
Do we need to add this to `requirements.txt`?

Reply:
Yes, caught again!